app/src/main/java/in/ac/bits_pilani/goa/ard/utils/AHC.java

```diff
@@ -19,7 +20,7 @@
     /**
      * Default Log tag for project.
      */
-    public static final String TAG = APPLICATION_ID;
+    public static final String TAG = "MAC_ARD";
```

Comment:
I thought about the SHORT_TAG that you used, but then again you are logging with a non app id tag.. we we are basically back at the same thing..